modules/rostests/apitests/ntdll/precomp.h

```diff
@@ -11,5 +11,7 @@
 #include <apitest_guard.h>
 #include <ndk/ntndk.h>
 #include <strsafe.h>
+#include <winbase.h>
+#include <winreg.h>
```

Comment:
Alternatively they can be placed in the .c file that actually needs them, i.e. RtlGetNtProductType.c .